Not launching after update

### Steps to reproduce

After the latest update, one of my suwayomi servers is not launching.

```
Suwayomi data location inside the container: /home/suwayomi/.local/share/Tachidesk
LD_PRELOAD=/opt/catch_abort.so /home/suwayomi/.local/share/Tachidesk/bin/kcef/libcef.so
18:06:54.664 [Thread-0] INFO suwayomi.tachidesk.server.ServerSetup -- Starting Android Main Loop
18:06:54.842 [main] INFO suwayomi.tachidesk.server.ServerSetup -- Running Suwayomi-Server v2.0.1864
18:06:54.975 [main] INFO suwayomi.tachidesk.server.util.AppMutex -- Mutex status is clear, Resuming startup.
18:06:55.168 [main] INFO suwayomi.tachidesk.server.database.DBManager -- Using H2 database version 1.4
18:06:55.257 [main] INFO de.neonew.exposed.migrations.RunMigrations -- Running migrations on database jdbc:h2:/home/suwayomi/.local/share/Tachidesk/database
18:06:55.375 [main] INFO de.neonew.exposed.migrations.RunMigrations -- Database version before migrations: 49
18:06:55.375 [main] INFO de.neonew.exposed.migrations.RunMigrations -- Running migration version 50: FixHandlingOfTooLongPageImageUrls
18:06:55.830 [main] ERROR suwayomi.tachidesk.server.ServerSetup -- unhandled exception
org.jetbrains.exposed.exceptions.ExposedSQLException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Constraint "UC_PAGE" not found; SQL statement:
ALTER TABLE PAGE DROP CONSTRAINT UC_PAGE; [90057-200]
	at org.jetbrains.exposed.sql.statements.Statement.executeIn$exposed_core(Statement.kt:99)
	at org.jetbrains.exposed.sql.Transaction.exec(Transaction.kt:292)
	at org.jetbrains.exposed.sql.Transaction.exec(Transaction.kt:269)
	at org.jetbrains.exposed.sql.Transaction.exec(Transaction.kt:235)
	at org.jetbrains.exposed.sql.Transaction.exec(Transaction.kt:205)
	at org.jetbrains.exposed.sql.Transaction.exec$default(Transaction.kt:201)
	at de.neonew.exposed.migrations.helpers.SQLMigration.run(SQLMigration.kt:20)
	at de.neonew.exposed.migrations.RunMigrationsKt.runMigrations$lambda$10$lambda$9(runMigrations.kt:50)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.inTopLevelTransaction$run(ThreadLocalTransactionManager.kt:355)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.inTopLevelTransaction$lambda$11(ThreadLocalTransactionManager.kt:402)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.keepAndRestoreTransactionRefAfterRun(ThreadLocalTransactionManager.kt:410)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.inTopLevelTransaction(ThreadLocalTransactionManager.kt:401)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.transaction$lambda$5(ThreadLocalTransactionManager.kt:310)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.keepAndRestoreTransactionRefAfterRun(ThreadLocalTransactionManager.kt:410)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.transaction(ThreadLocalTransactionManager.kt:259)
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.transaction(ThreadLocalTransactionManager.kt:237)
	at de.neonew.exposed.migrations.RunMigrationsKt.runMigrations(runMigrations.kt:49)
	at de.neonew.exposed.migrations.RunMigrationsKt.runMigrations$default(runMigrations.kt:26)
	at suwayomi.tachidesk.server.database.DBManagerKt.databaseUp(DBManager.kt:46)
	at suwayomi.tachidesk.server.database.DBManagerKt.databaseUp$default(DBManager.kt:39)
	at suwayomi.tachidesk.server.ServerSetupKt.applicationSetup(ServerSetup.kt:365)
	at suwayomi.tachidesk.MainKt.main(Main.kt:14)
	at suwayomi.tachidesk.MainKt.main(Main.kt)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Constraint "UC_PAGE" not found; SQL statement:
ALTER TABLE PAGE DROP CONSTRAINT UC_PAGE; [90057-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:41)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.jetbrains.exposed.sql.statements.jdbc.JdbcPreparedStatementImpl.executeUpdate(JdbcPreparedStatementImpl.kt:53)
	at org.jetbrains.exposed.sql.Transaction$exec$2.executeInternal(Transaction.kt:244)
	at org.jetbrains.exposed.sql.statements.Statement.executeIn$exposed_core(Statement.kt:97)
	... 22 common frames omitted
```

Downgrading to v2.0.1862 works.

### Expected behavior

It should launch

### Actual behavior

It is not

### Suwayomi-Server version

v2.0.1864 

### Server operating system

Unraid

### Server Desktop Environment

_No response_

### Server JVM version

idk

### Used client name

Suwayomi

### Client version

idk

### Used web browser

Chrome, but not the problem

### Client operating system

Docker

### Other details

ghcr.io/suwayomi/tachidesk:preview

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [x] I have written a short but informative title (ideally less than ~100 characters).
- [x] I have tried the troubleshooting guide described in [README.md](https://github.com/Suwayomi/Suwayomi-Server?tab=readme-ov-file#troubleshooting-and-support)
- [x] I have updated to the **[latest version](https://github.com/suwayomi/suwayomi-server/releases/latest)**.
- [x] I have filled out all of the requested information in this form, including specific version numbers.
- [x] I understand that **Suwayomi does not have or fix any extensions**, and I **will not receive help** for any issues related to sources or extensions.

--- a/server/src/main/kotlin/suwayomi/tachidesk/server/database/migration/M0050_FixHandlingOfTooLongPageImageUrls.kt
+++ b/server/src/main/kotlin/suwayomi/tachidesk/server/database/migration/M0050_FixHandlingOfTooLongPageImageUrls.kt
@@ -20,7 +20,7 @@ class M0050_FixHandlingOfTooLongPageImageUrls : SQLMigration() {
             GROUP BY INDEX, CHAPTER
         );
             
-        ALTER TABLE PAGE DROP CONSTRAINT UC_PAGE;
+        ALTER TABLE PAGE DROP CONSTRAINT IF EXISTS UC_PAGE;
         ALTER TABLE PAGE ADD CONSTRAINT UC_PAGE UNIQUE (INDEX, CHAPTER);
         
         ALTER TABLE PAGE ALTER COLUMN IMAGE_URL VARCHAR; -- the default length is `Integer.MAX_VALUE`

--- a/server/src/main/kotlin/suwayomi/tachidesk/server/database/migration/M0050_FixHandlingOfTooLongPageImageUrls.kt
+++ b/server/src/main/kotlin/suwayomi/tachidesk/server/database/migration/M0050_FixHandlingOfTooLongPageImageUrls.kt
@@ -13,6 +13,13 @@ import de.neonew.exposed.migrations.helpers.SQLMigration
 class M0050_FixHandlingOfTooLongPageImageUrls : SQLMigration() {
     override val sql: String =
         """
+        DELETE FROM PAGE
+        WHERE ID NOT IN (
+            SELECT MIN(ID)
+            FROM PAGE
+            GROUP BY INDEX, CHAPTER
+        );
+            
         ALTER TABLE PAGE DROP CONSTRAINT UC_PAGE;
         ALTER TABLE PAGE ADD CONSTRAINT UC_PAGE UNIQUE (INDEX, CHAPTER);
         

